src/app/legacy/containers/StoryPromo/utilities/index.js

```diff
@@ -4,11 +4,15 @@ import {
   MEDIA_ASSET_PAGE,
 } from '#app/routes/utils/pageTypes';
 
+// MAP is now either a Media Asset Page or a Media Article Page
 export const isMap = item => {
   const isCpsTypeMap = pathOr(null, ['cpsType'], item) === MEDIA_ASSET_PAGE;
   const hasMedia = pathOr(false, ['media'], item);
+  const isOptimoMediaPromo =
+    pathOr(false, ['contentType'], item) === 'Video' ||
+    pathOr(false, ['contentType'], item) === 'Audio';
```

Comment:
This could be simplified a little bit..

```suggestion
  const contentType = pathOr(null, ['contentType'], item);
  const isOptimoMediaPromo =  contentType === 'Video' || contentType === 'Audio';
```